src/main/java/net/javadiscord/javabot/systems/user_preferences/model/UserPreference.java

```diff
@@ -0,0 +1,17 @@
+package net.javadiscord.javabot.systems.user_preferences.model;
+
+import lombok.Data;
+
+/**
+ * Data class which represents a single user preference.
+ */
+@Data
+public class UserPreference {
+	private long userId;
+	private int ordinal;
```

Comment:
Maybe use the `Preference` enum here (if this is possible with the repository)

Reply:
yeah, but just saving the ordinal takes up less space instead of saving it as a string

Reply:
I didn't men saving the string in the DB. I meant using the `Preference` object here and still saving the ordinal to the DB.

Reply:
Oh, so you mean just merging the enum with the model class. Will do that!

Reply:
![image](https://user-images.githubusercontent.com/48297101/182050039-926e17d4-29ce-4450-98c1-5b7275c2c847.png)
sike

Reply:
No, I meant having the enum as an attribute of the model class and just saving the ordinal value in the DB.

Reply:
oh yeah